internal/datastore/postgres/watch.go

```diff
@@ -86,17 +86,19 @@ func (pgd *pgDatastore) Watch(
 					return
 				}
 
-				select {
-				case updates <- changeToWrite:
-				default:
-					errs <- datastore.NewWatchDisconnectedErr()
-					return
-				}
+				for _, changeToWrite := range changesToWrite {
+					changeToWrite := changeToWrite
 
-				currentTxn = revision
-			}
+					select {
+					case updates <- &changeToWrite:
```

Comment:
Add a comment below here saying "Nothing else to do"; while `select` doesn't allow fallthrough, it felt that way, so better to make it explicit IMO